fietsenrek/templates/base.html

```diff
@@ -0,0 +1,12 @@
+{% load staticfiles %}
+
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  {% block head %} {% endblock %}
+</head>
+<body>
+    {% block content %} {% endblock %}
+</body>
+</html>
```

Comment:
You're right and I'll have to look at the authentication again as I used a library that takes care of it all and doesn't provide an API.